### ISSUE TITLE
Cannot craft luminous crafting table or resonant wand

### ISSUE THREAD

**Issue by ranma862** [HellFirePvP/AstralSorcery#259]
Using version 1.6.1.

I find a large shrine, go down to the floating crystal and place the regular crafting table. The beam of light shines on it, but when I place the items for a luminous crafting table or resonant wand, no finished product shows up in the output.

I tried placing a luminous crafting table outside in creative mode, waited until nightfall and the starlight bar fills up, but neither one can be crafted there either. I've tried removing CraftingTweaks and CraftTweaker2 but it still doesn't work.

**Comment by HellFirePvP** [comment]
Pack name or modlist?

**Comment by ranma862** [comment]
AI Improvements |   | 0.0.1b3
Astral Sorcery |   | 1.6.1
AutoRegLib |   | 1.3-13
Bad Wither No Cookie - Reloaded |   | 1.12
Bed Bugs |   | 1.0.1
Better Than Mending |   | 1.1.1
BetterFps |   | 1.4.6
Bill of Materials |   | 0.1.2
BTFU continuous rsync incremental backup |   | 2.3
Chisel and Bits |   | 14.4
CodeChicken Lib |   | 3.1.1.288
COFH Core |   | 4.3.3.11
CoFH World |   | 1.0.0.6
Combined Potions |   | 1.1-beta3
Cooking for Blockheads |   | 6.3.2
Crafting Tweaks |   | 8.1.4
CraftTweaker |   | 4.0.2
Dimensional Ores |   | 1.0.1
Doggy Talents |   | 1.14.1.214
DynmapForge |   | 2.6
Embers |   | 0.221
Ender Crop |   | 1.3.4
Equal Dragons |   | 1
Expanded BoneMeal |   | 1.11
Extra Alchemy |   | 0.3.4.1
Extra Rails |   | 1.3.0
Farming for Blockheads |   | 3.1.3
FoamFix |   | 0.7.4
Gentle Harvest |   | Build 1
GraveStone |   | 1.8.4
Hwyla |   | 1.8.19-B33
Immersive Engineering |   | 0.12 - 70
Immersive Petroleum |   | 1.1.2
Iron Chests |   | 7.0.31.818
Journey Map |   | 5.4.9
Just a Few Fish |   | 1.7
Just Enough Items |   | 4.7.4.81
Just Enough Resources |   | 0.8.3.23
Lapis Stays in the Enchanting Table |   | 1.1.2-beta2
Level Up |   | 1.1.3
Mantle |   | 1.3.1.18
McJtyLib |   | 2.4.4
MCMultiPart |   | 2.2.2
Mining Lib |   | 1.0.1
Mod Tweaker |   | 4.0.1
More Shearables |   | 1.2.3
Mouse Tweaks |   | 2.8
MTLib |   | 3.0.0
Natura |   | 4.3.0.10
Natures Compass |   | 1.5.0
Neat |   | 1.4-15
Nether Portal Fix |   | 5.3.4
NetherEx |   | 2.1.11.12
No Mob Spawning on Trees |   | 1.2.1
Notes |   | 1.1.1
Obsidian Plates |   | 1.3.0
OverLord |   | 2.4.3
Pam's HarvestCraft |   | 1.12c
Potion Icon Remover |   | 4.0.0
Quark |   | 1.2-104
Quick Leaf Decay |   | 1.2.3
Redstone Flux |   | 2.0.1.2
RedstonePlusPlus |   | 1.2b
RF-Lux |   | 0.1.1
Roots |   | 2-0.025
Rustic |   | 0.4.1
Shadowfact's Forgelin |   | 1.5.1
ShadowMC |   | 3.8.0
Simple Gravel Ores |   | 1.7b
The Lost Cities |   | 0.0.19beta
Thermal Cultivation |   | 0.1.0.5
Thermal Dynamics |   | 2.3.3.10
Thermal Expansion |   | 5.3.3.15
Thermal Foundation |   | 2.3.3.10
Third Person Everything |   | 1.2.0
Tinkers Construct |   | 2.7.2
TrueType Font Replacement |   | 1.8.0
Underp Hangables |   | 1.0.0
Waila Harvestability |   | 1.1.11
Wawla |   | 2.5.239
Wither Skeleton Tweaks |   | 2.2.2
Xnet |   | 1.3.5

**Comment by ranma862** [comment]
I believe we found the compatibility issue, with RedstonePlusPlus, since removing it fixed the problem. From their modpage:

"  Improved Crafting Tables
Is now a TileEntity, so it will keep items rather then dropping them."

I'm guessing that made the crafting table not respond properly to recipes, or to the light beam from the floating crystal. For our part, we're probably going to remove RedstonePlusPlus from our server. Let me know if you want any further details on the testing we did or if a fix would be outside your scope. Thank you again for this fun mod!
